Sources/GEOSwift/GEOS/GeometryConvertible+GEOS.swift

```diff
@@ -463,6 +479,20 @@ public extension Collection where Element: GeometryConvertible {
     }
 }
 
+/// Buffer end cap styles
+public enum EndCapStyle: Int32 {
```

Comment:
not sure about naming `EndCapStyle` or `BufferEndCapStyle`?

Reply:
Let's go with `BufferEndCapStyle` since GEOS also use a `Buf` prefix.

+1 to Hashable & Sendable

Let's also remove Int32 as a raw type and instead add a non-public method to define a mapping from the swift cases to the corresponding GEOS values. The rationale here is that the numeric value is just an implementation detail and isn't really important in the Swift API.

Also, the implementation should be able to refer to the GEOS cases symbolically.

Reply:
I can't use GEOS enum values here https://github.com/GEOSwift/geos/blob/f510e634c822116fca615064d889300dba40d761/Sources/geos/public/geos_c.h#L3404-L3410 it expects Int32 but I can cast them to Int32 if that's okay

Reply:
Casting makes sense to me.